.github/workflows/deploy.pr.yml

```diff
@@ -9,11 +9,15 @@ env:
 jobs:
   deploy:
     name: Deploy Pull Request
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
```

Comment:
Is this required for this PR? If so can we add it to the readme? ;)

Reply:
I changed this because GithubActions wasn't working properly yesterday. Since last week ubuntu-latest uses Ubuntu 20 instead of Ubuntu 18, but I think that it was a general failure in GithubActions

Reply:
Now, github actions changes are kept to the minimum, just to fix small issues